app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -9,9 +9,7 @@ def linkedin
     # You need to implement the method below in your model (e.g. app/models/user.rb)
     @user = User.from_omniauth(auth_hash)
 
-    render json: @user
-
-    # TODO: reroute user based on signup/signin
+    render json: 'all done'
```

Comment:
Nice, so you confirmed that it needed a render statement, but that the actual payload doesn't matter?